app/models/join_team_player.rb

```diff
@@ -0,0 +1,2 @@
+class JoinTeamPlayer < ApplicationRecord
+end
```

Comment:
Missing relations with other models

this table should be a joining table so:

```suggestion
class JoinTeamPlayer < ApplicationRecord
  belongs_to :team
  belongs_to :user
end
```